test/failures_testsuite/base_test.py

```diff
@@ -75,7 +75,7 @@ def tearDownClass(cls):
         """
         self = cls()
         self._delete_directory(directory='tmp')
-        self.s3.failures.tlog_up()
+        #self.s3.failures.tlog_up()
```

Comment:
not sure why did u remove this ?

Reply:
it didn't work and raise error , islam need to edit it

Reply:
I removed it